shared/checks/oval/installed_env_has_pam_package.xml

```diff
@@ -31,7 +31,7 @@
     <linux:object object_ref="obj_env_has_pam_installed" />
   </linux:dpkginfo_test>
   <linux:dpkginfo_object id="obj_env_has_pam_installed" version="1">
-    <linux:name>pam</linux:name>
+    <linux:name>libpam-runtime</linux:name>
```

Comment:
I think you'll also need to map `pam` to `libpam-runtime` in a `package-override` for Ubuntu and Debian products.
Otherwise the remediations will still check for `pam`.
Something similar to:
https://github.com/ComplianceAsCode/content/blob/96cbe2466cc11cd9de20c6ed7061ae2a54e1ac60/sle15/product.yml#L27

Reply:
Aha, ty :)